modules/agent-install-dns-none.adoc

```diff
@@ -0,0 +1,172 @@
+:_content-type: CONCEPT
+[id="agent-install-dns-none_{context}"]
+= Platform "none" DNS requirements
+
+In {product-title} deployments, DNS name resolution is required for the following components:
+
+* The Kubernetes API
+* The {product-title} application wildcard
+* The control plane and compute machines
+
+Reverse DNS resolution is also required for the Kubernetes API, the control plane machines, and the compute machines.
+
+DNS A/AAAA or CNAME records are used for name resolution and PTR records are used for reverse name resolution. The reverse records are important because {op-system-first} uses the reverse records to set the hostnames for all the nodes, unless the hostnames are provided by DHCP. Additionally, the reverse records are used to generate the certificate signing requests (CSR) that {product-title} needs to operate.
```

Comment:
Lots of abbreviations in the first sentence. Should any of them be spelled out on first usage?